### ISSUE TITLE
Error while uploading to openneuro that is not detected by the bids validator 

### ISSUE THREAD

**Issue by munoztd0** [bids-standard/legacy-validator#1156]
Hello, I don't get why I get this error ONLY in Open Neuro but not when I run the bids-validator ? Also I shouldn't be able to upload the dataset with an error right? And moreover I don't even get this error what could it be can you point me in the direction to fix it if it's not just a bug?

![image](https://user-images.githubusercontent.com/43644805/105834636-057cd900-5fcb-11eb-9711-569142858b09.png)
 and the raw data is:

{
 "trial_phase": "Wether it is ITI or trial phase",
 "trial": "Counting number of trial",
 "condition": "Which condition: CS+, CS-, Baseline or None",
 "n_grips": "Number of grips during the trial",
 "rewarded_response": "The response that was rewarded on this trial",
 "phase":   {
    "Reminder": "Same as instrumental phase (2 time windows rewarded)",
    "Partial_Extinction": "Only one time window rewarded",
    "PIT": "No time window rewarded AKA exctinction"
  }

}

**Comment by effigies** [comment]
When you say "run the bids-validator", do you mean on the website, or do you have it installed via `npm`? If the latter, it may be a version issue.

**Comment by munoztd0** [comment]
I run it via docker. But also the weird thing is that the open neuro cli (via npm) runs the bids-validator each time you want to upload and there I also don't get any error (just minor warnings) but the error only appears on Openneuro website..

**Comment by nellh** [comment]
Do you get the same result with the standalone web version of the bids-validator? https://bids-standard.github.io/bids-validator/

**Comment by munoztd0** [comment]
I can't use the standalone web version the data is on a server and it's too big.

**Comment by effigies** [comment]
I strongly suspect you're using an older version of the validator. `events.json` validation was added in #1076 and released in 1.5.8.

In any event, please see [Tabular files](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#tabular-files) and [Task events](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html) to fix your JSON. It should be something more like:

```JSON
{
  "trial_phase": {
    "Description": "Wether it is ITI or trial phase"
  },
  "trial": {
    "Description": "Counting number of trial"
  },
  "condition": {
    "Description": "Which condition: CS+, CS-, Baseline or None"
  },
  "n_grips": {
    "Description": "Number of grips during the trial"
  },
  "rewarded_response": {
    "Description": "The response that was rewarded on this trial"
  },
  "phase": {
    "Levels": {
      "Reminder": "Same as instrumental phase (2 time windows rewarded)",
      "Partial_Extinction": "Only one time window rewarded",
      "PIT": "No time window rewarded AKA exctinction"
    }
  }
}
```

**Comment by munoztd0** [comment]
Well then it means the docker is not up to date ? Because is just run your "run -ti --rm -v /path/to/data:/data:ro bids/validator /data "  command

Anyhow thanks for your answer I'm going to change that!

**Comment by effigies** [comment]
You may also need to docker pull the latest version of the image. Docker will not check for a new version if there is a matching image cached locally.

**Comment by munoztd0** [comment]
Mhh sorry I didn't know docker "cached" the version, my bad.
